cp1404practicals/Prac_05/emails.py

```diff
@@ -0,0 +1,25 @@
+def main():
+    """Create dictionary of emails and names."""
+
+    emails_and_names = {}
+    get_email = input("Email: ")
```

Comment:
Not sure if "get_email" is a suitable name for this variable, to me it sounds more like a function name.